test/test_ops.py

```diff
@@ -1622,7 +1622,6 @@ def clone_and_perform_view(input, **kwargs):
                         self.assertEqual(tensor.grad, cloned1_tensor.grad)
 
     @ops(ops_and_refs, allowed_dtypes=(torch.cfloat,))
-    @skipIfTorchInductor("Inductor does not support complex dtype yet")
```

Comment:
do we know why this impacted complex dtype tests?

Reply:
I'm not sure what the complex dtype issue was originally but I think it was already fixed. `skipIfTorchInductor` is a bit of a heavy hammer, it skips all variants of a test even if only a subset is failing under inductor...

I thought the fact that the `test_neg_view_cuda` tests are now passing was kinda weird though. I'm surprised it's *just* the CUDA tests, and that the corresponding CUDA tests for `test_conj_view` and `test_neg_cong_view` weren't similarly affected. Do you want me to find the root cause? Might take a while

Reply:
It wasn't anything to do with complex - it was just this logic [here](https://github.com/pytorch/pytorch/blob/437b1aae4a693540a9da9c1cbca8f9b22a38a8cd/test/test_ops.py#L1550) to create the tensor that affected the tests